src/feature-flags.ts

```diff
@@ -1,115 +1,200 @@
 import { getApiClient, GitHubApiDetails } from "./api-client";
+import { CodeQL } from "./codeql";
 import { Logger } from "./logging";
 import { RepositoryNwo } from "./repository";
 import * as util from "./util";
 
-export interface FeatureFlags {
-  getValue(flag: FeatureFlag): Promise<boolean>;
+export interface FeatureEnablement {
+  getValue(feaature: Feature, codeql?: CodeQL): Promise<boolean>;
```

Comment:
```suggestion
  getValue(feature: Feature, codeql?: CodeQL): Promise<boolean>;
```

(+autogenerated code)